src/logic/collectibles/sources/Gnosis.ts

```diff
@@ -57,7 +56,7 @@ class Gnosis {
     return {
       address: mainAssetAddress,
       description: asset.name,
-      image: asset.logoUri || NFTIcon,
+      image: asset.logoUri,
```

Comment:
Should we not keep `NFTIcon` as a fallback in these places just in case?

Reply:
It's now in the JSX: https://github.com/gnosis/safe-react/pull/3331/files#diff-bc221ad54541a2428464bbb779b7ba97a79027c5a516d907b78a2122eaa0d815R125

Reply:
Inline styles 👀